internal/private/private.go

```diff
@@ -55,6 +55,7 @@ type ImageDestinationInternalOnly interface {
 	// It is available only if SupportsPutBlobPartial().
 	// Even if SupportsPutBlobPartial() returns true, the call can fail, in which case the caller
 	// should fall back to PutBlobWithOptions.
```

Comment:
The “in which case …” clause is now incorrect, and conflicts with the added text below.

(Also please update the copies of the comment in implementations.)